front/app/containers/Admin/projects/project/general/components/ProjectFolderSelect/ProjectFolderSelect.test.tsx

```diff
@@ -20,72 +17,156 @@ jest.mock('api/project_folders/useProjectFolders', () => () => {
   return { data: projectFolders };
 });
 
-jest.mock('api/me/useAuthUser');
+const mockUserData = makeUser({
+  roles: [{ type: 'admin' }],
+  highest_role: 'admin',
+});
+
+jest.mock('api/me/useAuthUser', () => () => ({ data: mockUserData }));
 
 describe('ProjectFolderSelect', () => {
-  it('should render', () => {
+  it('renders successfully', () => {
```

Comment:
Nice! Thanks for taking care of the tests. We now prefer e2e tests over unit tests but since this already existed, good to have it up to date.